app/src/main/java/bankingapp/CreateAccount.java

```diff
@@ -1,12 +1,9 @@
 package bankingapp;
 
-import java.sql.Connection;
-import java.sql.SQLException;
 import java.util.Random;
-import java.util.Scanner;
 
 /**
- * @author Fredrick. Class CreateAccount that creates a new bank account for a user. The user is required to provide
+ * @author Fredrick Kamau. Class CreateAccount that creates a new bank account for a user. The user is required to provide
```

Comment:
Text changes generally deserve a separate commit, so you don't have to roll them back if your code turns out to be bad. It sucks having to re-do stuff because something unrelated broke in the same commit.

And, since we're all doing our best, we never know whether our code is "good" while we're writing it.